### DISCUSSION TITLE
tryyy out one of my sans fights

### DISCUSSION THREAD

**Discussion by Nightingale08** [Jcw87/c2-sans-fight#123] [General]
Timeline Rush (easy)
[Timeline Rush.csv](https://github.com/Jcw87/c2-sans-fight/files/7740230/Timeline.Rush.csv)

Timeline Shredder (very hard)
[Timeline Shredder.csv](https://github.com/Jcw87/c2-sans-fight/files/7740231/Timeline.Shredder.csv)

**Comment by thelift112** [comment]
Alright, I'll try it out. I'll try the shredder. Okay this is very hard. Okay this is very good... But now I'll continue working on my own stuff...

**Reply by Nightingale08** [reply]
thank you btw its updated to 1.1.0, i added practice "mode" the attacks are in the "practice" folder (which means you can see each attack in the battle), heres the official game page of my timeline shredder: https://gamejolt.com/games/SHRED/672636 (its version 1.1.0).

**Reply by Nightingale08** [reply]
Heres the updated version from gamejolt:
[Timeline Shredder 1.1.0 (custom attack).zip](https://github.com/Jcw87/c2-sans-fight/files/8103953/Timeline.Shredder.1.1.0.custom.attack.zip)

also how are you doing its been a long time.

**Comment by wille360creator** [comment]
hey i need help what is the file name for when sans is thrashing you around the room and the bones are a bit bigger
